document/README.md

```diff
@@ -42,6 +44,8 @@ The rest of these instructions assume you are in the directory where is README i
 cd spec/document
 ```
 
+### Prerequisites
```

Comment:
Nit: Seems more logical to make this section the first one.

Reply:
Done!